### ISSUE TITLE
Clean up the Figma file

### ISSUE THREAD

**Issue by Parisajf** [hackforla/tdm-calculator#1494]
### Overview
We need to clean up and organize the figma file to make it easy for all users to find different parts of the design

### Action Items
- [ ] Making sure the page names are indicating the part of the project, having scattered designs all over one page ore centric and updating the names,maybe adding guidelines for non-designer users to be able to find what they need.

### Resources/Instructions
REPLACE THIS TEXT -If there is a website which has documentation that helps with this issue provide the link(s) here.

**Comment by Biuwa** [comment]
**Please provide an update:**

1. **Progress**_(What is the current status of your project? What have you completed, and what is left to do)_:
2. **Blockers**_(Difficulties or errors encountered)_:
3. **Availability**_(How much time will you have this week to work on this issue)_:
4. **ETA**_(When do you expect this issue to be completed)_:
5. **Pictures** _(if necessary)(Add any pictures that will help illustrate what you are working on)_:

**Comment by Biuwa** [comment]
@limjoyce 

**Please provide an update:**

1. Progress(What is the current status of your project? What have you completed, and what is left to do):
2. Blockers(Difficulties or errors encountered):
3. Availability(How much time will you have this week to work on this issue):
4. ETA(When do you expect this issue to be completed):
5. Pictures (if necessary)(Add any pictures that will help illustrate what you are working on)

**Comment by marlenamellody** [comment]
Progress: 

- Implemented a workflow for managing GitHub issues and drafting designs in Figma under the [resources](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=4041-12658&t=ZcdMlhQtqEA5c7Sn-1) section. 

- Updated naming conventions for pages under drafts and handoff: (#issue number - status) the title of the issue : name of designer

- Archived past issues

- Organized screens within previous handoff pages to establish a template for addressing new issues

**Comment by marlenamellody** [comment]
Progress: I created a new [Figma](https://www.figma.com/design/96Id2ddhbEAyyCF0WaIScV/TDM-Calculator-Handoff?node-id=115-770&t=uvNYKYOJiJ22ED8f-1) file dedicated to the design system and all screens for handoff, while archived designs will remain in the [original](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=4041-12650&t=jc9iwwvVlpDkwqno-1) file. Some of these screens will require updates to align with the current components.

Blockers: Is there anything else that needs to be included in the new figma file? Also, there might be a 3-page limit in the new file due to the project being on a starter figma plan.

Availability: 4 hours this week

ETA: 1-2 weeks

Images of design system and all pages included in the new file:

<img width="391" alt="Image" src="https://github.com/user-attachments/assets/83c0af06-cf50-40a9-b118-d290bc6a091a" />
<img width="612" alt="Image" src="https://github.com/user-attachments/assets/606525bd-49d4-4be9-b9af-b45446cad6e5" />
<img width="901" alt="Image" src="https://github.com/user-attachments/assets/5bbe1fa0-ea3b-45fe-be26-dd95a3a0ac23" />

**Comment by thetanmancan** [comment]
Hey @marlenamellody. I hope you had a nice winter vacation. Since we're back in action as of this week, please provide an update for this issue using the following format by Tuesday night. Thank you!

1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by marlenamellody** [comment]
**Progress:** I created additional pages for admin-facing screens and security-facing screens in the TDM Calculator Handoff file. I've also added screenshots of the site to the "all pages overview". Some screens (security page, submit snapshot) don't exist in Figma and need to be created. 

**Blockers:** 
- The TDM Calculator Handoff file with the additional pages is currently in the [DRAFTS](https://www.figma.com/design/cndwYJou6x5Sz0dGVVPgwT/TDM-Calculator-Handoff?node-id=0-1&t=3oSlcXXlzxdQxqwb-1) of the main TDM file. I'm not sure how to move this to the main file. 

**Availability:** 8 hours this week

**ETA:** ~2 weeks



<img width="240" alt="Image" src="https://github.com/user-attachments/assets/5309fd8f-2dc8-4f51-82ed-d39a18eb5c2b" />

<img width="1431" alt="Image" src="https://github.com/user-attachments/assets/cbadf5f6-1f09-45ff-a8f1-98676e842543" />

**Comment by thetanmancan** [comment]
Hi @marlenamellody. Could you give an update of your progress from this week?

Also, when posting images, Nilakshi advised using the following format:

<details><summary>Explanation of image</summary>

![file name](attached image)
</details>

The reason for using the details-arrow-twirl-down function (I don't know what it's actually called) is that it prevents a clutter of images in the Comments section. If there are a lot of displayed images in the Comments section, it requires users to have to scroll down a lot, which is not user-friendly. If you disagree with using that function, though, let me know. Thanks.

**Comment by marlenamellody** [comment]
Hi @thetanmancan, that sounds good - will keep that in mind for future updates. 

**Progress:** I've updated the screens with approved components from the design system. I noticed that side panel uses the font Oswald, which is not in our current style guide. 

**Blockers:** Same as last week - Need to move TDM Calculator Handoff file from [drafts](https://www.figma.com/design/cndwYJou6x5Sz0dGVVPgwT/TDM-Calculator-Handoff?node-id=0-1&t=vQ8kd2RG6sXTHyTW-1) to main [TDM calculator figma](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?m=auto&t=5wUeDIuwGTmqk5Dl-6) file. 

**Availability:** 8 hours this week

**ETA:** 1 week.

**Comment by marlenamellody** [comment]
**Progress:** No major updates this week. Some pages will need to be updated with approved designs (create project page 3-error, submit snapshot, security)

**Availability:** 8 hours this week

**ETA:** I think I'm finished with this issue for now. Need to gather feedback to determine if any changes are needed.

**Comment by thetanmancan** [comment]
@marlenamellody I spoke to Bonnie. Right now the Figma handoff file shows the default setting for each page in our system, but it needs to include everything in our system, including tooltips, modals, workflow information, etc. Essentially, the stakeholders need any information that will enable them to make changes to our system if they want to develop it further in the future after we hand it off to them.

I realize that that's a lot to include, especially considering we haven't finished designing or developing it. You might want to ask Bonnie about this, but should we move this to the Ice Box for the time being? This is a Should Have, and it's dependent on a lot of other design work being finished.

**Comment by marlenamellody** [comment]
@thetanmancan I agree. I will move this to the ice box for now and add the related dependencies.
